### PR TITLE
Health check endpoint

### PR DESCRIPTION
**Pull request by MartinBasti**
Signed-off-by: Martin Bašti <mbasti@redhat.com>

### CODE REVIEW

File: omps/api/v1/health.py [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+from flask import jsonify
+import requests
+from requests.exceptions import RequestException
+
+from . import API
+from omps.errors import KojiError
+from omps.quay import get_cnr_api_version
+from omps.koji_util import KOJI
+
+logger = logging.getLogger(__name__)
```
Comment (csomh) [inline omps/api/v1/health.py]: `logger` is not used here. Can it be removed?

Comment (MartinBasti) [reply omps/api/v1/health.py]: hmm, maybe I should rather log errors from koji/quay

Comment (csomh) [reply omps/api/v1/health.py]: That could be also done, but I would suggest to keep it in the response, too.

Comment (MartinBasti) [reply omps/api/v1/health.py]: Added

**Pull request merged**